pennylane/measurements/mid_measure.py

```diff
@@ -74,7 +74,7 @@ def func():
     >>> func()
     tensor([1., 0.], requires_grad=True)
 
-    Mid circuit measurements can be manipulated using the following dunder methods
+    Mid circuit measurements can be manipulated using the following dunder methods:
```

Comment:
I'm not a huge fan of the phrase "dunder method" in public docs since users may not be too familiar with how Python works behind-the-scenes. Maybe just "arithmetic operators" would be clearer here?